examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -225,30 +333,34 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
     function setInfoControlCountMap() {
       // Interaction function
       function highlightFeature(leafletEvent) {
-        var layer = leafletEvent.target;
-        layer.setStyle($scope.styles.hoverStyle);
-        if (!L.Browser.ie && !L.Browser.opera) {
-          layer.bringToFront();
+        if (cloudberry.parameters.maptype == 'countmap'){
+          var layer = leafletEvent.target;
+          layer.setStyle($scope.styles.hoverStyle);
+          if (!L.Browser.ie && !L.Browser.opera) {
+            layer.bringToFront();
+          }
+          $scope.selectedPlace = layer.feature;
```

Comment:
What does this function do?

Reply:
It changes the color of a polygon when the mouse is hovering on it.